ouroboros-network-api/src/Ouroboros/Network/NodeToClient/Version.hs

```diff
@@ -110,17 +110,17 @@ nodeToClientCodecCBORTerm v = CodecCBORTerm {encodeTerm, decodeTerm}
     where
       encodeTerm :: NodeToClientVersionData -> CBOR.Term
       encodeTerm NodeToClientVersionData { networkMagic, query }
-        | v >= NodeToClientV_16
+        | v >= NodeToClientV_15
```

Comment:
Should the tests in `ouroboros-network-protocols/testlib/Ouroboros/Network/Protocol/Handshake/Test.hs` also be updated to refer to `NodeToClientV_16`?

Reply:
You're absolutely right!

Reply:
Well, it should be `V_15`, but other than that you're right :)

Reply:
Fixed.